src/compiler/prioritization/priority-levels.md

```diff
@@ -0,0 +1,81 @@
+- Regressions where code still compiles but does something different than it used to do (dynamic semantics have changed)
+  - Mitigating conditions that may lower priority:
+    - If code uses feature that is explicitly not specified (e.g. `std::vec::Vec` docs state order in which it drops its elements is subject to change)
+- Feature-gated features accessible without a feature gate
+  - Mitigating conditions that may lower priority:
+    - If the pattern is VERY unlikely
+- Soundness holes with real-world implications
+  - Mitigating conditions that may lower priority:
+    - Soundness holes that are difficult to trigger
+    - Soundness holes that have been around for a very long time may be critical, but typically require
+    - Soundness holes that will not affect stable, e.g. if the hole makes use of a gated unstable feature.
+- Diagnostic regressions where the diagnostic is very common and the situation very confusing
+- ICEs for common scenarios or code patterns
+  - Mitigating conditions that may lower priority:
+    - If the code that triggers the ICE also triggers compilation errors, and those errors are emitted before the ICE
+    - If the code in question makes use of unstable features, particularly if the ICE requires a feature gate
+
+A P-critical issue will receive the most attention. It must be assigned one or several people as soon as possible, and the rest of the team should do their best to help them out if/when applicable.
+
+## P-high
+
+`P-high` issues are issues that need attention from the compiler team, but not to the point that they need to be discussed at every meeting.
+They can be `P-critical` issues that have a mitigating condition as defined above, or important issues that aren't deemed blockers.
+
+Because there are too many `P-high` issues to fit in every compiler meeting, they should rather be handled asynchronously by the Prioritization WG, in order to help them move forward. They can still occasionally be brought up at meetings when it is deemed necessary.
+
+The effectiveness of the Prioritization WG will be a direct consequence of our ability to draw the line between `P-critical` and `P-high` issues. There shouldn't be too many `P-critical` issues that compiler meetings become unmanageable, but critical issues shouldn't get lost in the list of P-high issues.
+
+P-high issues are issues the teams will mostly work on. We want to make sure they're assigned, and keep an eye on them.
+
+## P-medium and P-low
+
+`P-medium` refer to issues that aren't a priority for the team, and that will be resolved in the long run. Eg issues that will be fixed after a specific feature has landed.
+
+`P-low` refer to issues issue that the compiler team doesn't plan to resolve, but are still worth fixing.
+
+P-medium issues won't be our focus. 
+They are issues we would mentor someone interested in fixing. 
+They will remain in this state until someone complains, a community member fixes it, or it gets fixed by accident.
+
+---
```

Comment:
Do the formatting starting here make sense?. I'm not sure what's the intention, not saying that is bad :smile: just wondering if these are leftovers